README.md

```diff
@@ -1,12 +1,7 @@
-# Kuksa Feeders and Providers
-![kuksa.val Logo](./doc/img/logo.png)
+# THIS REPOSITORY HAS BEEN ARCHIVED, see [issue 174](https://github.com/eclipse/kuksa.val.feeders/issues/174) for details.
 
-*Note: This is a legacy repository. Most providers has been moved to new repositories!*
-
-
-Name | Description
----- | -----------
-[Replay](./replay)             |[KUKSA Server](https://github.com/eclipse/kuksa.val/tree/master/kuksa-val-server) replay script for previously recorded files, created by running KUKSA Server with `--record` argument **Note: Replay is deprecated and will reach End-of-Life 2024-12-31!**
+The Eclipse KUKSA project is reorganizing the repositories it used, and most content of this repository has now been moved to new repositories
```

Comment:
Just a small thought, could/should this be "has reorganized" because by the time of merging and archival the reorganization finishes. 
But keeping it as it is works for me as well.

Reply:
Just another thought. Could we point to a concrete repository which replaces this one? If not the general pointer to the org should be sufficient